batch/batch/resource_usage.py

```diff
@@ -82,6 +112,58 @@ def memory_usage_bytes(self) -> Optional[int]:
                 return int(f.read().rstrip())
         return None
 
+    def overlay_storage_usage_bytes(self) -> int:
+        return shutil.disk_usage(self.container_overlay).used
+
+    def io_storage_usage_bytes(self) -> int:
+        if self.io_volume_mount is not None:
+            return shutil.disk_usage(self.io_volume_mount).used
+        return 0
+
+    def is_attached_disk(self) -> bool:
+        return self.io_volume_mount is not None and os.path.ismount(self.io_volume_mount)
```

Comment:
nit: could probably just make this a private field instead of a method and then just call `os.path.ismount` once (not that it would really matter for performance).